pom.xml

```diff
@@ -22,8 +22,9 @@
     <properties>
         <revision>1.38</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.150.1</jenkins.version>
+        <jenkins.version>2.249.3</jenkins.version>
         <java.level>8</java.level>
+        <!--        <slf4j.version>1.7.26</slf4j.version>-->
```

Comment:
Accidentally left in?

```suggestion
```

Reply:
resolved in 393fcc5eb12afdb0f08dd39e4813c4433a254678